h5bp/ssl/policy_modern.conf

```diff
@@ -18,7 +18,22 @@
 # https://nginx.org/en/docs/http/ngx_http_ssl_module.html
 
 ssl_protocols TLSv1.2;
+
+# To enable TLS 1.3, make sure NGINX >=1.13 and OpenSSL >=1.1.1 are installed.
+#ssl_protocols TLSv1.2 TLSv1.3;
+
 ssl_ciphers EECDH+CHACHA20:EECDH+AES;
 # (1)
 ssl_ecdh_curve X25519:prime256v1:secp384r1:secp521r1;
 ssl_prefer_server_ciphers on;
+
+# Supported by NGINX >=1.15.3. Enables TLS 1.3 0-RTT, allows for faster resumption of TLS sessions.
+#
+# Warning: requests sent within early data are subject to replay attacks.
+#
+# To protect against such attacks at the application layer, the $ssl_early_data variable should be used:
+#   proxy_set_header Early-Data $ssl_early_data;
+#
+# The application should return response code 425 (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/425) for form submissions, XHR, searches, anything that could contain user supplied data.
```

Comment:
This looks very interesting but too early to add it, even more if it implies some application requirements.

Reply:
Also, please move comments in the file header and respect line max length.